src/java/org/apache/cassandra/index/sai/disk/v1/V1OnDiskFormat.java

```diff
@@ -215,7 +216,7 @@ public boolean validatePerSSTableComponents(IndexDescriptor indexDescriptor, boo
                 if (checksum)
                     SAICodecUtils.validateChecksum(input);
                 else
-                    SAICodecUtils.validate(input);
+                    SAICodecUtils.validate(input, Version.BA);
```

Comment:
Should we make it a config?

Reply:
I moved it to Version.VECTOR_EARLIEST, we don't change earliest via config so let's follow the same pattern.

Reply:
works for me